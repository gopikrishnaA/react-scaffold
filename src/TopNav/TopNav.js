import React, { Component } from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components'

const Grid = styled.div`
  display: grid;
  height: 56px;
  border-radius: 0;
  align-items: center;
  grid-template-columns: auto 1fr auto auto;
  grid-template-areas: ' Logo . ToolBar User';
  background-color: ${p => p.theme.palette && p.theme.palette.background.header};
  padding: 0 0 0 60px;
`

class TopNav extends Component {
  render() {
    
    return <Grid elevation={3}>{this.props.children}</Grid>
  }
}

TopNav.propTypes = {
  children: PropTypes.array
}
export default TopNav
