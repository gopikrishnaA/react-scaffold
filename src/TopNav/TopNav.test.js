import React from 'react'
import renderer from 'react-test-renderer'
import TopNav from './TopNav'

it('renders correctly', () => {
  const tree = renderer.create(<TopNav />).toJSON()
  expect(tree).toMatchSnapshot()
})
