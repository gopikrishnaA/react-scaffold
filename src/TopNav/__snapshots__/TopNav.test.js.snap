// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation3-6 MuiPaper-rounded-2 sc-bdVaJa ejWOPl"
/>
`;
